lib/kitchen/provisioner/chef_zero.rb

```diff
@@ -76,14 +76,18 @@ def run_command
 
         prefix_command(
           wrap_shell_code(
-            [cmd, *chef_client_args].join(" ").
+            [cmd, *chef_client_args, last_exit_code].join(" ").
```

Comment:
This feels a bit weird to me since we do some windows-specific stuff in this method call, but then also do some more in `#last_exit_code`.

Perhaps we should create a `#format_chef_command` method, pass `cmd` and `*chef_client_args` to it, and then it returns a string with the appropriate windows/PS-specific manipulations if needed.